### PR TITLE
Add alias metadata and image for Chop Sumo

### PR DESCRIPTION
**Pull request by kevinelliott**
Broken out from https://github.com/kukai-wallet/kukai/pull/208 this PR adds metadata and image support for Chop Sumo NFTs. This will merge into `staging` branch.

### CODE REVIEW

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -220,6 +220,13 @@ export const CONSTANTS: Constants = {
...
       thumbnailUrl: "../../../assets/img/alias/0xC0FACE-thumb.jpg",
       link: '', shouldDisplayLink: DisplayLinkOption.None
+    },
+    'ChopSumo': {
+      name: "Chop Sumo",
+      address: ["KT1Wm4Cegd7wW6MYxEhg6AH5iFX48y65nWvv"],
+      thumbnailUrl: "../../../assets/img/alias/chopsumo.png",
+      link: "https://chopsumo.xyz", shouldDisplayLink: DisplayLinkOption.None,
+      category: ["collectibles"]
     }
```
Comment (NozyKid) [inline src/environments/environment.prod.ts:230]: address: ["KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP"],
      thumbnailUrl: "../../../assets/img/alias/0xC0FACE-thumb.jpg",
      link: '', shouldDisplayLink: DisplayLinkOption.None
    },
    'CyberKidz Club': {
      name: "CyberKidz Club",
      address: ["tz1Z5yLks3yX4vk8QR7AsCto3GmgaJnx7dRA"],
      thumbnailUrl: "../../../assets/img/alias/cyberkidzclub.png",
      link: "https://cyberkidzclub.xyz", shouldDisplayLink: DisplayLinkOption.None,
      category: ["collectibles"]
      description: "In 2420, only 6666 kids survived the destruction of the global ecosystem caused by the excessive minting of dirty NFTs on proof of work blockchains. CyberKidz Club."

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -220,6 +220,13 @@ export const CONSTANTS: Constants = {
       address: ["KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP"],
       thumbnailUrl: "../../../assets/img/alias/0xC0FACE-thumb.jpg",
```
Comment (NozyKid) [inline src/environments/environment.prod.ts:221]: cyberkidzclub

**Review by iamdakotaquinn — COMMENTED** [review]
This adds some alias metadata and images for CyberKidz Club, a Tezos NFT Collection By @StickHen_ and @crunchytez.

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -960,6 +967,7 @@ export const TRUSTED_TOKEN_CONTRACTS = [
   'KT1Ak8AFA54waFVMyPXNE925cUaQjFEqxuYN',//HOLLOWS
   'KT1BkStQej7MwnkWhCV2tUanETeN25zc7ADe',//TacoNFT premint tokens
   'KT1Mf8Pqn6NMt8VGuVaySohvSvQQpvcM37eZ',//TacoNFT
-  'KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP'//0xC0FACE
+  'KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP',//0xC0FACE
+  'KT1Wm4Cegd7wW6MYxEhg6AH5iFX48y65nWvv'//ChopSumo
 ];
```
Comment (NozyKid) [inline src/environments/environment.prod.ts:972]: 'KT1DuZFNv6JwwA7kS8RXvoQiANm3L4dfn6qP'//CyberKidz Club

**Pull request merged**